dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEnrollmentAnalyticsManager.java

```diff
@@ -335,6 +335,20 @@ else if ( params.isOrganisationUnitMode( OrganisationUnitSelectionMode.CHILDREN
                 + params.getProgramStatus().stream().map( p -> "'" + p.name() + "'" ).collect( joining( "," ) ) + ") ";
         }
 
+        if ( params.hasCreatedBy() )
+        {
+            sql += hlp.whereAnd() + " createdby in ("
+                + params.getCreatedBy().stream().map( username -> "'" + username + "'" ).collect( joining( "," ) )
+                + ") ";
+        }
+
+        if ( params.hasLastUpdatedBy() )
+        {
+            sql += hlp.whereAnd() + " lastupdatedby in ("
+                + params.getLastUpdatedBy().stream().map( username -> "'" + username + "'" ).collect( joining( "," ) )
```

Comment:
I think it's better to use
```
org.hisp.dhis.analytics.util.AnalyticsSqlUtils#encode
```
to avoid SQL injection

Reply:
Good point. Done.